### PR TITLE
Add Windows documentation

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

File: doc/wincalibration.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    Calibration generated in /tmp/calibrate_compass.g8oM/mag_config.yaml.
+    Restart ROS service to begin using saved calibration.
+
+
+Once complete, you will be prompted to enter the user password in order to save the new calibration
+into ``$ROS_ETC_DIR``. Once this is done, restart the ROS service to begin using the new calibration:
+
+.. code-block:: bash
+
+    sudo service ros restart
```
Comment (seanyen) [inline doc/wincalibration.rst:65]: Is this something needed to be translated to Windows command?

File: doc/winsetup.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+4. Install ROS on Windows using the `ROS Wiki instructions`_.
+5. Please follow the instructions closely, as ROS is sensitive to the operating environment. 
+
+This will install the development platform on the Jackal, which allows you to develop your robotics solutions. Once you've 
+finalized your development, you can build a deployment package and deploy your solutions using Chocolatey. 
+
+If you see any errors, please `get in touch`_ and we'll see if we can get you sorted out.
+
+.. _download a trial: https://www.microsoft.com/en-us/evalcenter/evaluate-windows-10-enterprise
+.. _ROS Wiki instructions: https://wiki.ros.org/Installation/Windows)
```
Comment (seanyen) [inline doc/winsetup.rst:30]: Removing the extra `)` at the end?

File: doc/wincalibration.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Calibration should be performed when jackal is first received, and any time the platform is modified by adding
```
Comment (lilustga) [inline doc/wincalibration.rst:67]: Perhaps move this to the top where seasonal calibration is mentioned. Redundant having it in 2 places and better to have all cases where calibration are required at the beginning.

File: doc/wincalibration.rst (hunk 1 above)
Comment (ooeygui) [reply doc/wincalibration.rst:65]: Oops, I removed that block in another file. I'll add the startup commands.

**Pull request merged**